repository/m.json

```diff
@@ -66,6 +66,18 @@
 			]
 		},
 		{
+			"name": "Magento2 Snippets",
+			"details": "https://github.com/parthviroja/sublime-magento2-snippets",
+			"labels": ["mage", "magento", "magento2", "snippets", "theme", "module"],
+			"author" : "Parth M. Viroja",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```

Comment:
This needs to be changed to `"tags": true` and then you need to create a semver tag on the repo (e.g. 1.0.0).